src/RestClient.Net.UnitTests/IUrlProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿#if !NET45
+
+
+namespace RestClient.Net.UnitTests
+{
+    public interface IUrlProvider
+    {
+#pragma warning disable CA1055 // URI-like return values should not be strings
+        string GetUrl();
```

Comment:
@DrLeh this is a nice addition but I'd prefer that this return an `AbsoluteUrl` instead of a string. It's just that I'm trying to communicate the benefits of using `Urls` instead of passing around strings.

Reply:
This was just a mechanism to replicate my use case of reading a string in from something like a config file for the purposes of a unit test

Reply:
I understand.